src/styles/blocks/stars.scss

```
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
```

Combine with `background` shorthand property
```suggestion
  height: 16px;
  width: 16px;
  background: .....your solution;
```